test/queueservice_test.js

```diff
@@ -50,10 +50,11 @@ suite('queue/QueueService', function() {
   test('putDeadlineMessage, pollDeadlineQueue', async () => {
     var taskId      = slugid.v4();
     var taskGroupId = slugid.v4();
+    var schedulerId = 'fake-garbage-scheduler';
```

Comment:
Use a slugid instead... they are designed to fit these generic identifiers.. .and we reduce conflicts with tests running in parallel (we do have other conflicts)